.eleventy.js

```diff
@@ -0,0 +1,114 @@
+const fs = require('fs');
+const pluginRss = require('@11ty/eleventy-plugin-rss');
+const scMeetupDetails = require('./_includes/shortcodes/meetup-details');
+const filterFullDate = require('./_includes/filters/full-date');
+
+/**
+ * Add date properties to collections.
+ *
+ * @param  {object}   posts   Collection to add date properties to.
+ * @return {object}           Augmented posts collection.
+ */
+const addData = (posts) => posts.map((post) => {
+  const stat = fs.statSync(post.inputPath) || {};
+
+  post.dateCreated = stat.birthtime;
+  post.dateModified = stat.mtime;
+
+  return post;
+});
+
+module.exports = (eleventyConfig) => {
+  // PLUGIN: RSS feed
+  eleventyConfig.addPlugin(pluginRss);
+
+  // PASSTHRU: Copy the `assets` directory to the compiled site folder
+  eleventyConfig.addPassthroughCopy('assets');
+
+  // COLLECTION: Create meetup collection.
+  eleventyConfig.addCollection('meetups', (collection) => {
+    const posts = collection.getFilteredByGlob('./_meetups/**');
+
+    return addData(posts);
+  });
+
+  // COLLECTION: Create post collection.
+  eleventyConfig.addCollection('posts', (collection) => {
+    const posts = collection.getFilteredByGlob('./_posts/**');
+
+    return addData(posts);
+  });
+
+  // NUNJUCKS SHORTCODE: Format meeting details message block.
+  eleventyConfig.addShortcode('meetupDetails', scMeetupDetails);
+
+  // TRANSFORM: Add appropriate TARGET and REL to external links.
+  eleventyConfig.addTransform('external-link-rel', (content) => {
```

Comment:
i'd vote against this

Reply:
Are you voting against opening external links in new tabs/windows?

FWIW, I got used to adding the feature after product/technical writers kept demanding it. I can remove very easily.

Reply:
i am. i don't care that much, but since it's a Normal Website, if you navigate off the site and don't want to be there, navigating back will be fast and work well. feels like we're trying to keep people from leaving the page kinda.

Reply:
I'll add an issue to remove it, since I'm rather numb to the [arguments](https://medium.com/the-metric/links-should-open-in-the-same-window-447da3ae59ba) and usually rather rely on default browser behavior anyhow.

Reply:
added #114